prometheus/generic/setup-netapp-exporter/tasks/prereqs.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Ensure epel-release is installed
+  yum:
```

Comment:
same comment as the HP iLO PR:
https://github.com/redhat-cop/monitoring/pull/44#pullrequestreview-654767130